airflow/cli/commands/worker_command.py

```diff
@@ -28,6 +28,14 @@
 from airflow.utils.cli import setup_locations, setup_logging, sigint_handler
 
 
+def _serve_logs(env, skip_serve_logs=False):
+    """Starts serve logs process on port 80"""
```

Comment:
Remove the 80 since it's a configurable port (just """Starts process to serve task logs""")